systest/backup/common/utils.go

```diff
@@ -91,6 +102,221 @@ func CopyToLocalFs(t *testing.T) {
 	require.NoError(t, testutil.DockerCp(srcPath, copyBackupDir))
 }
 
+func RemoveContentsOfPerticularDir(t *testing.T, dir string) {
```

Comment:
we should remove this function now that we are not using it

Reply:
Resolved